scrapers/ScrapeWithURL.py

```diff
@@ -0,0 +1,46 @@
+import json
+import sys
+
+import py_common.graphql as graphql
+import py_common.log as log

```

Comment:
recently we started using
```suggestion
try:
    import py_common.graphql as graphql
    import py_common.log as log
except ModuleNotFoundError:
    print("You need to download the folder 'py_common' from the community repo! (CommunityScrapers/tree/master/scrapers/py_common)", file=sys.stderr)
    sys.exit()
```
so that the user can understand/notice if he didnt copy the required `py_common` folder from the repo